app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/views/fragments/DataCollectionFragment.kt

```diff
@@ -136,14 +142,19 @@ class DataCollectionFragment : Fragment() {
             drawer_layout.closeDrawer(GravityCompat.START)
         }
 
-        identifyResultViewModel.showPopupAttributeListEvent.observeEvent(viewLifecycleOwner) {
+        identifyResultViewModel.showPopupEvent.observeEvent(viewLifecycleOwner) {
             bottomSheetNavController.navigate(R.id.action_identifyResultFragment_to_popupAttributeListFragment)
```

Comment:
I think the action should be renamed to `action_identifyResultFragment_to_popupFragment`

Reply:
updated